src/gui/accountsettings.cpp

```diff
@@ -360,7 +360,29 @@ void AccountSettings::showConnectionLabel(const QString &message, QStringList er
         ui->connectionStatusLabel->setToolTip(QString());
     }
     ui->accountStatus->setVisible(!message.isEmpty());
```

Comment:
As we might have an overlap with https://github.com/owncloud/client/blob/master/src/gui/folderstatusmodel.cpp#L56-L56 is there a chance we can refactor the existing code?

Reply:
Yes, but it's also used for the systray icon, and as data item in the folder table model, so it's more "delicate". I think it would be better to do that in a separate PR.